### PR TITLE
kill G

### PR DESCRIPTION
**Pull request by maxtaco**
This won't pass circle yet, but maybe it'll pass jenkins.

Eliminate `libkb.G`

### CODE REVIEW

**Comment by maxtaco** [comment]
CI depends on https://github.com/keybase/kbfs/pull/1217

**Comment by maxtaco** [comment]
W00t w00t, it passed jenkins!!

File: go/chat/inboxsource.go [hunk 1]
```diff
@@ -214,6 +214,7 @@ func newBaseInboxSource(g *globals.Context, ibs types.InboxSource,
 func (b *baseInboxSource) notifyTlfFinalize(ctx context.Context, username string) {
 	// Let the rest of the system know this user has changed
 	finalizeUser, err := libkb.LoadUser(libkb.LoadUserArg{
+		Contextified:      libkb.NewContextified(b.G().ExternalG()),
```
Comment (patrickxb) [inline go/chat/inboxsource.go]: Wish we could enforce the use of libkb.NewLoadUserArg*(g,...*)

Comment (maxtaco) [reply go/chat/inboxsource.go]: Done.  Some other pr

**Comment by maxtaco** [comment]
Maybe hold off on this one?

**Comment by maxtaco** [comment]
Killed in favor of #8822

**Pull request closed without merging**